Tests/LibWeb/Screenshot/expected/border-radius-normalization-ref.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+    body {
+      margin: 0;
+      padding: 20px;
+      background-color: #f5f5f5;
+      font-family: sans-serif;
+    }
+
+    .test-case {
+      margin: 15px 0;
+      display: flex;
+      align-items: center;
+      gap: 15px;
+    }
+
+    .label {
+      width: 200px;
+      font-size: 14px;
+      color: #333;
+    }
+
+    .box {
+      background-color: #4CAF50;
+      border: 2px solid #333;
+    }
+
+    /* Float max values properly normalized to perfect circle */
+    .float-max-precision {
+      width: 300px;
+      height: 40px;
+      border-radius: 20px; /* Float max values normalized to height/2 for perfect pill shape */
+      background-color: #4CAF50;
+      border: 1px solid #333;
+    }
+
+    h2 {
+      font-size: 18px;
+      margin-bottom: 20px;
+      color: #333;
+    }
+  </style>
+</head>
+<body>
+  <h2>Border Radius Normalization Test</h2>
+
+  <div class="test-case">
+    <div class="label">Float max precision test:</div>
+    <div class="float-max-precision"></div>
+  </div>
+</body>
+</html>
```

Comment:
You've made a ref test, not a screenshot test. Which is better! So you can delete the PNG file and move this to `Tests/LibWeb/Ref` instead :-)

Reply:
Oops - thanks for the help! Moved this to `Tests/LibWeb/Ref` now and ran LibWeb tests successfully - should be good to go hopefully.